notebook/i18n/de/LC_MESSAGES/notebook.po

```diff
@@ -0,0 +1,416 @@
+msgstr ""
+"Wenn jede Zeile von Ausgabe wahr ist, wird sie ein JSON Objekt mit den "
+"Details von der Serverinformationsdatei sein."
+
+msgid "Currently running servers:"
+msgstr "Gegenwärtig Server ausführen:"
+
+msgid "Don't open the notebook in a browser after startup."
+msgstr "Öffnen Sie den Notizblock in einem Browser nicht nach Startup."
+
+msgid ""
+"DISABLED: use %pylab or %matplotlib in the notebook to enable matplotlib."
+msgstr ""
+"Unfähig gemacht: Verwenden Sie %pylab oder %matplotlib im Notizblock, um "
+"matplotlib zu ermöglichen."
+
+msgid "Allow the notebook to be run from root user."
+msgstr "Erlauben Sie, dass der Notizblock von Root geführt wird."
+
+msgid ""
+"The Jupyter HTML Notebook.\n"
+"    \n"
+"    This launches a Tornado based HTML Notebook Server that serves up an "
+"HTML5/Javascript Notebook client."
+msgstr ""
+"Die Jupyter HTML Notebook.n n dies startet einen tornadobasierten HTML "
+"Notizblockserver, der einen HTML5/ Javascript Notizblockkunden hinauf dient."
+
+msgid ""
+"Deprecated: Use minified JS file or not, mainly use during dev to avoid JS "
+msgstr ""
+"Missbilligt: Verwenden Sie minified JS Akte oder nicht vermeiden Sie während "
+"dev hauptsächlich gewöhnlich JS recompilation"
+
+msgid "Set the Access-Control-Allow-Credentials: true header"
+msgstr ""
+"Setzen Sie die Zugangskontrolle erlauben Sie Ausweispapiere: Wahre Überschrift"
+
+msgid "Whether to allow the user to run the notebook as root."
+msgstr "Ob, dem Benutzer zu erlauben, den Notizblock als Wurzel zu führen."
+
+msgid "The default URL to redirect to from `/`"
+msgstr "Die versäumte URL, zu der man umleiten kann, von"
+
+msgid "The IP address the notebook server will listen on."
+msgstr "Die IP Adresse der Notizblockserver wird zuhören auf."
+
+msgid ""
+"Cannot bind to localhost, using 127.0.0.1 as default ip\n"
+"%s"
+msgstr ""
+"Kann keine Bindung zu localhost mit Hilfe von 127.0.0.1 als versäumte ipn%s"
+
+msgid "The port the notebook server will listen on."
+msgstr "Der Hafen, auf dem der Notizblockserver zuhören wird."
+
+msgid ""
+"The number of additional ports to try if the specified port is not available."
+msgstr ""
+"Die Anzahl zusätzlicher Häfen, zu versuchen, wenn der angegebene Hafen nicht "
+"zur Verfügung steht."
+
+msgid "The full path to an SSL/TLS certificate file."
+msgstr "Der volle Pfad in eine SSL/ TLS Zertifikatakte."
+
+msgid "The full path to a private key file for usage with SSL/TLS."
+msgstr ""
+"Der volle Pfad zu einer privaten Schlüsseldatei für Gebrauch mit SSL/ TLS."
+
+msgid ""
+"The full path to a certificate authority certificate for SSL/TLS client "
+"authentication."
+msgstr ""
+"Der volle Pfad zu einem Zertifikatautoritätszertifikat für SSL/ TLS "
+"Kundenbestätigung."
+
+msgid "The file where the cookie secret is stored."
+msgstr "Die Akte, wo das Keksgeheimnis gespeichert wird."
+
+msgid "Writing notebook server cookie secret to %s"
+msgstr "Notizblockserverkeksgeheimnis auf %s schreiben"
```

Comment:
My German coworker is laughing uproariously at this translation. I think that while 'keks' is the translation of an edible cookie, an HTTP cookie is called a cookie.